src/api/users/users.schema.ts

```diff
@@ -2,4 +2,8 @@ import type { User, UserInput } from '@domain/models/users.model.js';
 
 export interface CreateUserRequestBody extends UserInput {}
 
-export interface CreateUserResponse extends User {}
+export interface UserResponse extends User {}
+
+export interface GetUserInput {
+  id: string;
+}
```

Comment:
```suggestion
export interface GetUserPathParams {
  id: string;
}
```

Só uma sugestão daqui que vc usou o sufixo `RequestBody` para os bodies de entrada, aqui como é um model de api, acho que faz sentido tbm um sufixo para o "tipo" do model.